src/coreclr/src/tools/crossgen2/crossgen2/Program.cs

```diff
@@ -313,7 +313,8 @@ private int Run()
                     }
                     CompilationBuilder builder = new ReadyToRunCodegenCompilationBuilder(typeSystemContext, compilationGroup, inputFilePath,
                         ibcTuning: _commandLineOptions.Tuning,
-                        resilient: _commandLineOptions.Resilient);
+                        resilient: _commandLineOptions.Resilient,
+                        parallelism: _commandLineOptions.Parallelism);
```

Comment:
I'd prefer to see this passed to the builder via the fluent interface pattern like the use of UseBackendOptions below.

Reply:
Sure. Is there a reason that ibcTuning and resilient shouldn't be moved as well? I see that input file path is used to get the input module in the constructor but those aren't.

Reply:
No, they really should be moved to.

Reply:
Thanks for making this change @SrivastavaAnubhav